src/components/Grid/Grid.js

```diff
@@ -51,8 +33,47 @@ Grid.propTypes = {
    * class name
    */
   className: PropTypes.string,
+  /**
+   * flavor: string or array of strings. Flavors: frame, vertical, reverse,
+   * vertical-reverse, align-center, align-spread, align-space, align-end,
+   * vertical-align-center, certical-align-end, vertical-stretch, pull-padded,
+   * pull-padded-medium, pull-padded-large,
+   */
+  flavor: PropTypes.oneOfType([PropTypes.oneOf([
+    'frame',
+    'vertical',
+    'reverse',
+    'vertical-reverse',
+    'align-center',
+    'align-spread',
+    'align-space',
+    'align-end',
+    'vertical-align-center',
+    'certical-align-end',
+    'vertical-stretch',
+    'pull-padded',
+    'pull-padded-medium',
+    'pull-padded-large',
+  ]), PropTypes.arrayOf(PropTypes.oneOf([
+    'frame',
+    'vertical',
+    'reverse',
+    'vertical-reverse',
+    'align-center',
+    'align-spread',
+    'align-space',
+    'align-end',
+    'vertical-align-center',
+    'certical-align-end',
+    'vertical-stretch',
+    'pull-padded',
+    'pull-padded-medium',
+    'pull-padded-large',
+  ]))]),
+  /*
+   * wrap variation
+   */
+  wrap: PropTypes.bool,
```

Comment:
This makes no sense from a user's perspective. Why is wrap a prop and other flexbox properties not?

Reply:
Because wrap is a variation `slds-wrap' and the others are flavors e.g. `slds-grid_align-spread`.